src/pages/VenuePage/VenuePage.tsx

```diff
@@ -172,15 +172,19 @@ const VenuePage: React.FC = () => {
 
   // useVenueAccess(venue, handleAccessDenied);
 
-  if (!user) {
-    return <Login formType="initial" />;
+  if (!venue || !venueId) {
+    return <LoadingPage />;
   }
 
   if (venueRequestStatus && !venue) {
     return <>This venue does not exist</>;
   }
 
-  if (!venue || !venueId || !profile) {
+  if (!user) {
+    return <Login formType="initial" venue={venue} />;
```

Comment:
`initial` is the default value, you can remove the prop `formType`

Reply:
Removed in https://github.com/sparkletown/sparkle/pull/1546/commits/3eaee127c14e82bae162ec5a42eeba2fdfcb888a